src/components/index.js

```diff
@@ -87,6 +103,13 @@ export default {
       }
     });
     Vue.mixin({ store });
+
+    //Helper to access data reference.
+    Vue.mixin({ methods:{ getScreenDataReference(customProperties = null, setter = null) {
```

Comment:
Is this helper used? seems it is not used anymore in your PR

Reply:
It was removed in this PR